what-is-software-development/study-like-a-programmer.md

```diff
@@ -0,0 +1,340 @@
+
+### Your Perseverence & Self-Care Skills
+
+#### We Need Little Breaks to Stay Focused
+
+Feeling like you lose focus over time? Our brains have evolved to focus on things for relatively short periods of time. We need little breaks to stay focused.
+
+Consider using our favorite time management technique, [Pomodoro Technique](https://en.wikipedia.org/wiki/Pomodoro_Technique), to help enable frequent breaks and frequent periods of flow!
+
+Encourage yourself and others to take breaks often.
+
+#### Sleep
+
+Your brain is probably going to be much less productive at fixing a bug on the 11th hour of work today than you will be if you get good sleep and tackle the same bug first thing in the morning.  Also, don't be afraid to take naps!
+
+#### Feel Like a Programmer!
+
+Sometimes the struggle comes from not solidifying your identity as a programmer. **You are a software developer!** Tell this to yourself out loud every day, if it helps.
+
+### !challenge
+
+* type: multiple-choice
+* id: 777d35be-f76c-442e-8b75-33499b5dbc87
+* title: Study Skills
+
+##### !question
+
+Which of the following is _not_ a category of studying skills that was covered above?
+
+##### !end-question
+
+##### !options
+
+* Question-Asking Skills
+* Review Skills
+* Perseverence & Self-Care Skills
+* Bottling-It-Up Skills
+
+##### !end-options
+
+##### !answer
+
+* Bottling-It-Up Skills
+
+##### !end-answer
+
+### !end-challenge
+
+## Activities and Techniques that are Tried and True
+
+- Feeling like classroom time is overwhelming? Consider: Look at the calendar and the lessons the night before, and pre-read the lectures
+- Feeling like reading material on programming is unnatural? Consider: Read this resource on [how to read programming books](http://www.pathsensitive.com/2018/01/the-benjamin-franklin-method-of-reading.html)
+- Feeling like you lose focus on the task at hand? Consider: Using physical paper and sticky notes to write down the specific task you are trying to accomplish
+- Feeling like you lose focus over time? Consider: Using a time management method like [Pomodoro Technique](https://en.wikipedia.org/wiki/Pomodoro_Technique) to help enable frequent breaks and frequent periods of flow
+- Finding that your knowledge and understanding level of vocabulary is lacking? Consider: Utilize flashcards for specific vocabulary, concepts, and syntax
+- Finding that you are craving more review in your own words? Consider: Making crib sheets (1-page review notes of the crucial knowledge) at the end of the week
+- Feeling isolated? Please consider having proactive communication with other students and staff! Staff is on your side for your success, and your proactive communication with staff can lead to opportunities and resources
+- Want a video series we recommend? Consider: ["Crash Course" on study skills](https://www.youtube.com/playlist?list=PL8dPuuaLjXtNcAJRf3bE1IJU6nMfHj86W)
```

Comment:
```suggestion
<div style="width=100px;">Are you feeling like... </div>| Consider |
---- | ---- |
...classroom time is overwhelming? | Look at the calendar and the lessons the night before, and pre-read the lectures
...reading material on programming is unnatural? | Read this resource on [how to read programming books](http://www.pathsensitive.com/2018/01/the-benjamin-franklin-method-of-reading.html)
...you lose focus on the task at hand? | Using physical paper and sticky notes to write down the specific task you are trying to accomplish
...you lose focus over time? | Using a time management method like [Pomodoro Technique](https://en.wikipedia.org/wiki/Pomodoro_Technique) to help enable frequent breaks and frequent periods of flow
...your knowledge and understanding level of vocabulary is lacking? | Utilize flashcards for specific vocabulary, concepts, and syntax
...you are craving more review in your own words? | Making crib sheets (1-page review notes of the crucial knowledge) at the end of the week
... you're isolated? | Please consider having proactive communication with other students and staff! Staff is on your side for your success, and your proactive communication with staff can lead to opportunities and resources

Want a video series we recommend? Consider: ["Crash Course" on study skills](https://www.youtube.com/playlist?list=PL8dPuuaLjXtNcAJRf3bE1IJU6nMfHj86W)

```

I put this into a table to make it a little easier to read. Learn preview wasn't working for me so I'm not quite sure if the inline style worked, but it's there if you'd like to keep it.